cfgov/core/templatetags/richtext.py

```diff
@@ -0,0 +1,23 @@
+from django import template
+
+
+register = template.Library()
+
+
+@register.filter
+def richtext_isempty(value):
+    """
+    Returns True if a value is None, an empty string, or empty paragraph tags
+
+    Working around known issue https://github.com/wagtail/wagtail/issues/3111.
+    Draftail RichTextBlocks passed in require accessing the `source` attribute
+    for the comparison.
+    """
+
+    return any([
+        value is None,
+        value == '',
+        value == '<p></p>',
+        hasattr(value, 'source') and value.source == '',
+        hasattr(value, 'source') and value.source == '<p></p>',
+    ])
```

Comment:
I suggested some of this code, but now I think we could avoid some repetition:

```python
blank_values = ['', None, '<p></p>']
return any([
    value in blank_values,
    hasattr(value, 'source') and value.source in blank_values,
])
```

This would also cover the case of value.source being None, Not sure that case exists, but can't hurt to check.

Reply:
Done in 8c1196a, thanks for the suggestion!